### DISCUSSION TITLE
2.6.14 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#3402] [讨论]
· 由于 MCBBS 关站，移除相关内容
　　移除 MCBBS 镜像下载源
　　移除 Mod/整合包 下载页面的 MCBBS 跳转（#3385，@liubanlaobanzhang）
　　移除特别鸣谢页面的 MCBBS 项（#3391，@misividkoukou）
　　移除统一通行证的查看简介按钮（#3397，@Ad-closeNN）
　　移除帮助库中的 MCBBS 相关内容（LTCatt/PCL2Help#350，by @WForst-Breeze）
　　将自定义主页教程中的 MCBBS 相关内容替换为其他网站
· MC 下载优化
　　优化仅使用 BMCLAPI 时的下载策略
　　优化使用 OptiFine 中文站下载时片大小出错的下载策略
　　添加是否验证 SSL 证书的选项（#3018）
　　使用 HTTP 长连接，减少握手次数（#3293，@youzi-2333）
· 完善功能描述与提示信息
　　修改覆盖第三方登录设置时的提示文本（#3045，@LiteCat0905）
　　拖拽安装单个 Mod 的提示信息不严谨（#3177，@xhahello）
　　修改功能隐藏中邮箱显示一项的描述（#3292，@youzi-2333）
　　为启动前执行命令设置添加运行目录描述（#3332，@youzi-2333）
　　更新特别鸣谢中的部分描述
· 崩溃分析支持识别新版 Fabric 处置建议（#3228、#3231，@xhahello，by @zkitefly）
· 为启动前执行命令和自定义游戏标题添加了 {date} 替换标记
· 帮助库添加禁人塔导致无法登录的描述（LTCatt/PCL2Help#348，by @youzi-2333）
· 更新 2024 年 1 月赞助者列表，感谢大家的支持！
· 更新 Mod 中文名数据库
· 修复了以下 16 个 Bug（包含 1 个重要 Bug）
　　**[重要] 无法全新安装 MC（#3383，@Tactsohg）**
　　部分系统环境下无法启动游戏（#1108，by @00ll00）
　　可能无法检测到特定 Java（#1496，@RawDiamondMC）
　　有文件冲突时无法加载 Mod 列表（#3092，@WEDFVV）
　　部分过长的 Mod 名会遮挡关闭按钮（#3094，@JingHai-Lingyun）
　　搜索时拖拽安装 Mod 导致出错（#3124，@w2w2e2wq）
　　指定单个登录方式时的图标不统一（#3139，@Harry-hq）
　　删除 Mod 时可能出错（#3151，@FanrklinCN）
　　第三方登录地址删除 / 导致链接不完整（#3194，@N1ngYu）
　　在 Mod、整合包 页面翻页时可能出错（#3208，@gi199）
　　低版本 Java 无法安装 Forge 1.20.3+（#3211，@vcvcOTZ）
　　部分文本框允许输入过长以至于卡顿的内容（#3242，@rumengchuxing062）
　　背景图片、音乐可能读取到 Windows 文件（#3365，@ling-314）
　　启动前执行命令的 {time} 标记没有得到替换（#3395，@Ad-closeNN）
　　未对服务器公告信息进行校验（@Silverteal）
　　无法从 BMCLAPI 下载部分文件

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.6.14'>2.6.14</a>.</em>

**Comment by Ad-closeNN** [comment]
第一次等到版本更新qwq 龙猫早点睡吧
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/db47fb1c-c15b-4e24-b5e9-6cdaf0b49c2d)
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/c0200a23-2498-4963-9f66-1ff87be5d489)

**Comment by Silverteal** [comment]
阴间更新时间
~~话说我为什么会收到邮件~~
更新：瞎了，没看见@我

**Reply by Ad-closeNN** [reply]
因为你被龙猫@了
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/41632b8f-015e-4101-81b3-1c553ef625a3)

**Reply by Ad-closeNN** [reply]
坐等龙猫把PCL上传到服务器.jpg

**Comment by allMagicNB** [comment]
龙猫你再晚点更新恐怕1月和2月的高级赞助者都要一起放了

**Reply by Deep-Dark-Forest** [reply]
很抱歉，补票上的54.32档位无法被计入名单

**Comment by Harry-hq** [comment]
第一次等到活跃橙

**Comment by gi199** [comment]
干嘛
________________________________
发件人: Harry-hq ***@***.***>
发送时间: 2024年2月22日 18:58
收件人: Hex-Dragon/PCL2 ***@***.***>
抄送: gi199 ***@***.***>; Mention ***@***.***>
主题: Re: [Hex-Dragon/PCL2] 2.6.14 更新日志 (Discussion #3402)


第一次等到活跃橙

―

**Reply by allMagicNB** [reply]
？

**Comment by gi199** [comment]
干嘛
________________________________
发件人: allMagic ***@***.***>
发送时间: 2024年2月23日 20:18
收件人: Hex-Dragon/PCL2 ***@***.***>
抄送: gi199 ***@***.***>; Mention ***@***.***>
主题: Re: [Hex-Dragon/PCL2] 2.6.14 更新日志 (Discussion #3402)


？

―

**Comment by HanXiaozeng** [comment]
忘恩负义！怎么能把MCBBS在鸣谢名单中删除呢！

**Reply by Ad-closeNN** [reply]
没办法，要怪就怪它跑路咯

**Comment by HanXiaozeng** [comment]
也对，估计没钱了

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Ad_closeNN ***@***.***>
发送时间: Saturday, March 2, 2024 11:22:34 AM
收件人: Hex-Dragon/PCL2 ***@***.***>
抄送: Han_Xiaozeng ***@***.***>; Comment ***@***.***>
主题: Re: [Hex-Dragon/PCL2] 2.6.14 更新日志 (Discussion #3402)


没办法，要怪就怪它跑路咯

―
